Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs

```diff
@@ -305,5 +305,27 @@ public void Dispose()
                 _state.StateChanged -= HandleStateChanged;
             }
         }
+        private string _toDoFilter;
```

Comment:
Minor item. Add a new line between the closing brace and `private string _toDoFilter`.

Reply:
```suggestion

        private string _toDoFilter;
```